### PR TITLE
526 form endpoint

### PR DESCRIPTION
**Pull request by egarreau**

### CODE REVIEW

File: rakelib/swagger.rake [hunk 1]
```diff
@@ -6,7 +6,8 @@ namespace :swagger do
     raise IOError, 'No json-schema file provided' unless args[:json_schema_file]
     schema_path = Rails.root.join('spec', 'support', 'schemas', args[:json_schema_file])
     raise IOError, "No json-schema file at #{schema_path}" unless File.exist? schema_path
-    json = JSON.parse(schema_path)
+    schema = File.read(schema_path)
```
Comment (kreek) [inline rakelib/swagger.rake]: thanks for fixing 👍 I think `JSON.load` will both read and parse

File: app/swagger/schemas/rated_disabilities.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+               special_issues]
+        property :decision_code, type: :string, example: 'SVCCONNCTED'
+        property :decision_text, type: :string, example: 'Service Connected'
+        property :name, type: :string, example: 'Diabetes mellitus0'
+        property :effective_date, type: :datetime, example: '2018-03-27T21:00:41.000+0000'
+        property :rated_disability_id, type: :string, example: '0'
+        property :rating_decision_id, type: :string, example: '63655'
+        property :rating_percentage, type: :integer, example: '100'
+        property :related_disability_date, type: :datetime, example: '2018-03-27T21:00:41.000+0000'
+        property :special_issues, type: :array, example: 'TODO'
```
Comment (kreek) [inline app/swagger/schemas/rated_disabilities.rb]: is this left over from the swagger block generator?

File: spec/request/disability_compensation_form_request_spec.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  let(:token) { 'fa0f28d6-224a-4015-a3b0-81e77de269f2' }
+  let(:auth_header) { { 'Authorization' => "Token token=#{token}" } }
+
+  before do
+    Session.create(uuid: user.uuid, token: token)
+    User.create(user)
+  end
+
+  describe 'Get /v0/disability_compensation_form/rated_disabilities' do
+    context 'with a valid evss response' do
```
Comment (kreek) [inline spec/request/disability_compensation_form_request_spec.rb]: this could be in another PR but we need specs for the http error status codes listed in the EVSS swagger [400, 403, 500] to ensure they're mapping to the right vets-api codes

File: app/swagger/schemas/rated_disabilities.rb (hunk 2 above)
Comment (egarreau) [reply app/swagger/schemas/rated_disabilities.rb]: WHOOPS yup thanks
